server/aws-lsp-codewhisperer/src/language-server/proxy-server.ts

```diff
@@ -7,50 +7,23 @@ import { QNetTransformServerToken } from './netTransformServer'
 import { QChatServer } from './qChatServer'
 import { QConfigurationServerToken } from './configuration/qConfigurationServer'
 import { readFileSync } from 'fs'
-import { ConfigurationOptions } from 'aws-sdk'
 import { HttpsProxyAgent } from 'hpagent'
 import { NodeHttpHandler } from '@smithy/node-http-handler'
 
-const makeProxyConfig = () => {
-    let additionalAwsConfig: ConfigurationOptions = {}
-    const proxyUrl = process.env.HTTPS_PROXY ?? process.env.https_proxy
-
-    if (proxyUrl) {
-        const certs = process.env.AWS_CA_BUNDLE ? [readFileSync(process.env.AWS_CA_BUNDLE)] : undefined
-        const agent = new HttpsProxyAgent({
-            proxy: proxyUrl,
-            ca: certs,
-        })
-
-        additionalAwsConfig = {
-            httpOptions: {
-                agent: agent,
-            },
-        }
-    }
-
-    return additionalAwsConfig
-}
-
 export const CodeWhispererServerTokenProxy = CodewhispererServerFactory(credentialsProvider => {
```

Comment:
With this change we are able to remove this `*Proxy` factories - they don't do anything special now, and we have exports of duplicate "regular" factories already, for example here https://github.com/aws/language-servers/blob/479871ed7bce0343526e9c7b985784a4c744372b/server/aws-lsp-codewhisperer/src/language-server/codeWhispererServer.ts#L664-L669.

Only if we remove these, we'll need to make sure we consume correct exports in our distribution pipeline.

We can first migrate distribution process, and then remove these as follow-up.